src/commonMain/kotlin/io/github/detekt/sarif4k/SarifDataBindings.kt

```diff
@@ -339,13 +341,43 @@ data class Address (
  *
  * Key/value pairs that provide additional information about the version control details.
  */
-@Serializable
-data class PropertyBag (
+@Serializable(with = PropertyBag.Companion::class)
+@JvmInline
+value class PropertyBag(
```

Comment:
Should there be a comment on top of the class, similar to SarifSchema210 to call out manual changes for next auto-generation?

Reply:
I don't think so, there are a lot of other manual changes no documented. Also, the diff gives a lot of information.

Reply:
I changed my mind, it was an easy change so I did it.